doc/lua_api.txt

```diff
@@ -971,20 +971,64 @@ There are a bunch of different looking node types.
 Look for examples in `games/minimal` or `games/minetest_game`.
 
 * `normal`
+    * A node-sized cube.
 * `airlike`
+    * Invisible, uses no texture.
 * `liquid`
+    * The cubic source node for a liquid.
 * `flowingliquid`
+    * The flowing version of a liquid, appearing with various heights and
+      slopes.
 * `glasslike`
+    * Often used for partially-transparent nodes.
+    * Only external sides of textures are visible.
 * `glasslike_framed`
+    * All face-connected nodes are drawn as one volume within a surrounding
+      frame.
+    * The frame appearence is generated from the edge pixels of the first
+      texture specified in `tiles` (preferably 16x16 pixels).
```

Comment:
Why "preferably 16x16 pixels"? Does it not work correctly with anything else? As long as it works correctly with other sizes, this shouldn't be included as it's just your / the MTG preference, and this is general document that should apply for any game or mod no matter what texture size they favour.

Reply:
Yeah it's complex, will explain. The frame is created from the edge 1/16th of a texture, so other resolutions reate a messy result, will detail this.

Reply:
Duh, obviously other resolutions can be used, will explain.